localization/beluga_demo_bearing_localization/maps/ekuthon_playground_landmarks_map.yaml

```diff
@@ -1,34 +1,34 @@
 frame_id: map
 features:
-  - position: [-6 , -6 , 1.0]
+  - position: [5, 0, 1]
     category: 0
-  - position: [-6 , -2, 1.0]
+  - position: [5, 2.5, 1]
     category: 0
-  - position: [-6 , 0, 1.0]
+  - position: [5, 5, 1]
     category: 0
-  - position: [-6 , 2, 1.0]
+  - position: [2.5, 5, 1]
     category: 0
-  - position: [-6 , 6 , 1.0]
+  - position: [0, 5, 1]
     category: 0
-  - position: [-2, -6 , 1.0]
+  - position: [-2.5, 5, 1]
     category: 0
-  - position: [-2, 6 , 1.0]
+  - position: [-5, 5, 1]
```

Comment:
Any particular reason for reducing the size from 12x12 to  10x10?

Reply:
Aesthetics. Relocating them above walls lights them up.